src/stream/src/executor/aggregation/agg_impl/approx_count_distinct.rs

```diff
@@ -12,30 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! This module implements `StreamingApproxCountDistinct`.
```

Comment:
Still think that `Updateable` is more clear than `Deletable`...

Reply:
> Still think that `Updateable` is more clear than `Deletable`...

ok